spec/database_connection_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'database_connection'
+
+describe DatabaseConnection do 
+  let(:connection) { DatabaseConnection.setup('twitter_test') }
+  describe '.setup' do 
+    it 'should set up a database connection' do
+      expect(PG).to receive(:connect).with(dbname: 'twitter_test')
+      DatabaseConnection.setup('twitter_test')
+    end 
+  end 
+
+  describe 'connection' do 
+    it 'should know connection' do 
+      expect(connection).to eq connection
+    end 
```

Comment:
I'm not sure what this tests for? Is it that you're testing if the variable exists?